tests/station07.test.tsx

```diff
@@ -1,6 +1,24 @@
-// No tests for Station No.7.
-// Proceed to the meeting reservation.
-// TODO: ここのテストを書く
-// TODO: クリアしてほしい基準を決定→たたきを大木へ送る(→問題文からみて決定する)
+import { render } from '@testing-library/react'
+import * as React from 'react'
+import { createAsync } from './utils/createAsync'
+import { compareColor } from './utils/compareColor'
 
-test.skip('No tests for Station No.7', () => {})
+const { App } = require('../src/App') as { App: React.ComponentType<{}> }
+
+describe('<App />', () => {
+  it('<App /> has a <header>', async () => {
+    const res = await createAsync(<App />)
+    res.root.findByType('header')
+  })
+
+  it('#F5F5F5 is specified as the background color of <header>', async () => {
+    const { container } = render(<App />)
+    const header = container.querySelector('header')
+
+    if(!header){
+      throw new Error('<header> not exists')
+    }
```

Comment:
```suggestion
    if(!header) {
      throw new Error('a <header> does not exist')
    }
```

英文ミスとコーディングスタイルの修正です．エディタにPrettierプラグインを入れるのをオススメします．